manifests/pipecd/grafana-dashboards/control-plane/overview.json

```diff
@@ -531,7 +531,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum by (project) (grpcapi_create_deployment_total)",
+          "expr": "sum by (project) (round(increase(grpcapi_create_deployment_total[24h])))",
```

Comment:
I found some floating-point numbers in the results, and adding `round` behaves as expected.